feature-libs/asm/root/services/asm-enabler.service.ts

```diff
@@ -14,10 +20,25 @@ import { ASM_ENABLED_LOCAL_STORAGE_KEY } from '../asm-constants';
   providedIn: 'root',
 })
 export class AsmEnablerService {
+  private asmModuleInstance$: Observable<
+    NgModuleRef<any>
+  > = this.configInitializer.getStable('featureModules').pipe(
+    map((config: CmsConfig) => config.featureModules ?? {}),
+    switchMap((featureModulesConfig) =>
+      this.lazyModule.resolveModuleInstance(
```

Comment:
Just realized that this solution will only work for lazy loaded asm, it should also work for eager loading ootb, the solution here could be checking, if `eatureModulesConfig['asm']?.module` is defined, and if not, just return `of(undefined)` in the `swtichMap`.